.github/workflows/rollback.yml

```
@@ -3,6 +3,7 @@
 # Revert a release. `reason` is required (audit log).
 # Each platform job uses its own environment (rollback-{platform}) with required reviewers.
 name: Rollback
+run-name: "⏪ Rollback · ${{ inputs.version_tag || github.ref_name }}"
```

actionlint (1.7.12)

[error] property "version_tag" is not defined in object type {platform: string; reason: string; rollback_to_version: string}

(expression)